core/app/models/spree/line_item.rb

```diff
@@ -57,6 +57,13 @@ def amount
     def discounted_amount
       amount + promo_total
     end
+    deprecate discounted_amount: :final_amount_without_additional_tax, deprecator: Spree::Deprecation
+
+    # @return [BigDecimal] the amount of this item, taking into consideration
+    #   all non-tax adjustments.
+    def final_amount_without_additional_tax
+      amount + adjustments.reject(&:tax?).sum(&:amount)
```

Comment:
This probably needs to check `eligible?` (and a spec catching that would be great)

Reply:
👍 thanks. Added a spec as well.